app/src/test/java/edu/ucandroid/weathernotice/ExampleUnitTest.kt

```diff
@@ -46,23 +46,29 @@ class ExampleUnitTest {
         whenJSONDataAreReadAndParsed()
         thenResultsShouldContainBelize()
     }
+
+    @Test
```

Comment:
Added test annotations that were missing

https://junit.org/junit5/docs/current/user-guide/

Reply:
No need to annotate the given, when, and then methods; instead, annotate the method that invokes them.

As a matter of fact, you don't want to annotate these, as several of them don't contain asserts.  It will look like you have more tests than you actually do.